### ISSUE TITLE
Question: Can the github workflow be more generic and not only used with Azure ?

### ISSUE THREAD

**Issue by cmoulliard** [parodos-dev/workflow-software-templates#21]
## Question

The github flow which is created uses azure actions to set the k8s context and deploy the tekton resources. Such a workflow will only work on the Azure Cloud platform.

See: https://github.com/ch007m/onboarding/blob/main/.github/workflows/deploy.yaml#L18-L24

Could it be possible to only install such Tekton resources on azure if the user has selected Azure part of the template ?
Can such a flow be more generic and support to deploy the resources on an on-premises OCP cluster ?

**Comment by pkliczewski** [comment]
@dmartinol please take a look

**Comment by dmartinol** [comment]
@cmoulliard the azure action is actually generic and works on any OpenShift cluster, given that you provide it the proper configuration in the `set-context` step.
Instead, I would reword this issue to adopt the RH implementation, e.g. the `redhat-actions/oc-login` action.

**Comment by cmoulliard** [comment]
> the azure action is actually generic

Glad to hear that. Maybe add a line within the README to help the user ;-)
